commands/generate/generate.go

```diff
@@ -98,7 +98,7 @@ func (s *Spec) Exec(_ context.Context) error {
 
 	log.Trace().Msg("all handlers stopped, generating final report")
 
-	if s.PrintReport {
+	if s.PrintReport || report.HasFailures() {
```

Comment:
small issue: when a `experimental clone` fails to generate, no report was being shown. I think this was introduced in the commands refactor.

Reply:
got this when solving issues with the `config.LoadSubTree()`